repo2docker/app.py

```diff
@@ -122,6 +122,17 @@ def fetch(self, url, ref, checkout_path):
                                extra=dict(phase='failed'))
                 sys.exit(1)
 
+    def validate_image_name(self, image_name):
+        """
+        Validate that the image_name read by argparse contains only lowercase characters
+        :param image_name: string argument read by argparse
+        :return: test and return image_name if it only contains lowercase characters else raise error
```

Comment:
On this line I'd just say "unmodified image_name", and the fact that it tests for lowercase letters should be made clear from the first sentence of the docstring on line 127.

Reply:
Fixed